builtin/providers/dns/data_dns_a_record_set_test.go

```diff
@@ -34,23 +36,30 @@ func TestAccDataDnsARecordSet_Basic(t *testing.T) {
 			[]string{
 				"129.6.15.30",
 			},
+			"time-c.nist.gov",
 		},
 	}
 
-	var steps []resource.TestStep
-
 	for _, test := range tests {
-		ts := resource.TestStep{
-			Config: test.DataSourceBlock,
-			Check: resource.ComposeTestCheckFunc(
-				testCheckAttrStringArray(fmt.Sprintf("data.dns_a_record_set.%s", test.DataSourceName), "addrs", test.Expected),
-			),
-		}
-		steps = append(steps, ts)
+		recordName := fmt.Sprintf("data.dns_a_record_set.%s", test.DataSourceName)
+
+		resource.UnitTest(t, resource.TestCase{
```

Comment:
I just looked into it further, and I see now that UnitTest is just a wrapper around the Test to allow some Acceptance tests to run with Unit tests.

So, it is still an acceptance test, but by using this method, the three tests will run with the unit tests as well (which I think should be ok, unless we view them as having external dependencies due to the use of DNS).

wdyt?